test/test_verilog_iverilog_options.vader

```diff
@@ -0,0 +1,24 @@
+Before:
+  Save g:ale_verilog_iverilog_options
+  let g:ale_verilog_iverilog_options = ''
+
+After:
+  Restore
+  call ale#linter#Reset()
+
+Execute(Set Verilog iverilog linter additional options to `-y.`):
+  runtime! ale_linters/verilog/iverilog.vim
+
+  " Additional args for the linter
+  let g:ale_verilog_iverilog_options = '-y.'
+
+  call ale#Lint()
+
+  let g:run_cmd = ale_linters#verilog#iverilog#GetCommand(bufnr(''))
+  let g:matched = match(g:run_cmd, '\s' . g:ale_verilog_iverilog_options . '\s')
+
+  " match returns -1 if not found
+  AssertNotEqual
+  \   g:matched ,
+  \   -1 ,
```

Comment:
Use `AssertEqual` with the full command string instead. That's more reliable.

Reply:
Got it. I've changed the assertion to match the whole desired string.